tools/utils.ts

```diff
@@ -1,6 +1,9 @@
 import * as yargsParser from 'yargs-parser';
 import type * as Enquirer from 'enquirer';
 
+// This file uses the non-standard jsdoc tag @remarks
+/* eslint jsdoc/check-tag-names: ['error', { definedTags: ['remarks'] }] */
```

Comment:
same thing -> `@remarks` [is used by TSDoc](https://tsdoc.org/pages/tags/remarks/) and VScode for intellisense ( need to be discussed which way we wanna go )

Reply:
Just to clarify, this is _allowing_ the `@remarks` tag for this file only, because this is the only file that used it. If we wanted to allow it for all files, I can move this exception to the default config.

Reply:
I moved this exception to the default config to apply to all files